pkg/k8s/apis/cilium.io/v1/types.go

```diff
@@ -145,6 +153,12 @@ func (r *CiliumNetworkPolicy) Parse() (api.Rules, error) {
 	return retRules, nil
 }
 
+// GetControllerName returns the unique name for the controller manager.
+func (r *CiliumNetworkPolicy) GetControllerName() string {
+	name := k8sconst.GetObjNamespaceName(&r.ObjectMeta)
+	return fmt.Sprintf("%s (v1 %s)", k8sconst.CtrlPrefixPolicyStatus, name)
```

Comment:
Should the v1 in the string also be a parameter? or is it the version of the k8s meta thing? I noticed we use this function for both v1 and v2 CNP controllers.

Reply:
No, each version has its own `GetControllerName()`